### PR TITLE
Added Edge remote update topic and updated the graphics.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

**Comment by MWindel** [comment]
Hi Sharath, I am currently reviewing the PR but only for the language and for general understanding. Please get an expert (team member etc.) to review the content as well :)

**Comment by sharath-jk** [comment]
> Hi Sharath, I am currently reviewing the PR but only for the language and for general understanding. Please get an expert (team member etc.) to review the content as well :)

**Comment by sharath-jk** [comment]
@MWindel, I have got the approval for technical accuracy over an email. I cannot add the reviewer here in the PR. So the reviewer sent the approval over an email.

**Review by MWindel — APPROVED** [review]

File: content/edge/remote-connectivity-bundle/accessing-the-edge-appliance-remotely-through-vnc.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Step 3: Connecting to the Edge appliance using VNC
+
+To access and connect to the Edge appliance:
+
+1. In the {{< product-c8y-iot >}} tenant for your registered Edge appliance, add a remote access endpoint. See [Adding remote access endpoints](/cloud-remote-access/using-cloud-remote-access/#adding-remote-access-endpoints-via-vnc). You must use the same port number that is allocated for you.
+
+2. Connect to the endpoint. See [Connecting to endpoints](/cloud-remote-access/using-cloud-remote-access/#connecting-to-endpoints).
+
+The connection to the Edge appliance is established and the GUI appears for the Edge appliance. Right-click in the screen to open the desktop components **xterm** and **firefox**.
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/accessing-the-edge-appliance-remotely-through-vnc.md:45]: GUI > UI

File: content/edge/remote-connectivity-bundle/config-remote-connectivity.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: Configuring remote-connectivity using the UI
+layout: redirect
+---
+
+1. Log in to the {{< management-tenant >}}.
+
+	- Username: management/<*username*>
+	- Password: password provided during the installation
+
+2. Switch to the **Administration** application using the application switcher at the right of the top bar **<img class="Default" src="/images/icons/switcher-icon.png" alt="icon" style="display: inline; float: none">**.
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/config-remote-connectivity.md:12]: **Administration** > Administration

File: content/edge/remote-connectivity-bundle/config-remote-connectivity.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+3. Click **Edge** > **Remote Connectivity** in the navigator.
+
+4. Click **Enable remote connectivity** toggle to enable remote-connectivity.
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/config-remote-connectivity.md:16]: Click **Enable remote connectivity** toggle > Switch the **Enable remote connectivity** toggle or Activate the **Enable remote connectivity** toggle? Click sounds strange.

File: content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* devices (and more generically, managed objects)
+* events
+* alarms
+* measurements
+
+>**Important:** Data broker in {{< product-c8y-iot >}} Edge does not support synchronization of the [operations](/concepts/domain-model/#operations).
+
+To upload the data to a {{< product-c8y-iot >}} tenant account, you must first create a data connector in the Edge appliance and subscribe this connector in the tenant account. 
+
+A data connector describes the subset of the data that you would like to send to a destination tenant. For more information, see [{{< enterprise-tenant >}} > Using the data broker > Data connector](/users-guide/enterprise-tenant/#data-connectors).
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md:20]: add "in the *User* guide".

File: content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+4. Log in to the {{< product-c8y-iot >}} tenant account.
+
+5. In the tenant account, go to the Administration application. Click **Data broker** > **Data subscriptions** to subscribe the connector created in your Edge appliance.
+
+   Click **Add data subscription** and provide the security code. Click **Submit** and accept the subscription. See [{{< enterprise-tenant >}} > Using the data broker > Data subscriptions](/users-guide/enterprise-tenant/#data-subscriptions) in the *User guide*.
+
+   You can now navigate to the Device Management application or the Cockpit application. You will find a new “virtual group” with a specific icon showing the forwarded devices. The group will have the same name as your subscription. 
+
+For more information about sending and receiving data in {{< product-c8y-iot >}}, see [{{< enterprise-tenant >}} > Using the data broker](/users-guide/enterprise-tenant#data-broker).
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md:42]: add "in the User guide".

File: content/edge/remote-connectivity-bundle/edge-remote-reboot.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+weight: 40
+title: Restarting Edge remotely
+layout: redirect
+---
+
+The {{< product-c8y-iot >}} tenant allows you to remotely restart your Edge appliance. To do so:
+
+1. Log in to your {{< product-c8y-iot >}} tenant.
+
+2. Go to **Device Management**.
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/edge-remote-reboot.md:11]: **Device Management** > the Device Management application

File: content/edge/remote-connectivity-bundle/registering-the-edge-appliance-in-the-cumulocity-iot-tenant.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+weight: 20
+title: Registering the Edge appliance in the Cumulocity IoT tenant
+layout: redirect
+---
+
+Before performing these steps, ensure that you have configured the URL for the {{< product-c8y-iot >}} tenant in the Edge appliance.   
+
+1. Log in to your {{< product-c8y-iot >}} tenant.
+
+2. Go to **Device Management**.
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/registering-the-edge-appliance-in-the-cumulocity-iot-tenant.md:11]: see above

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**